content/en/docs/reference/access-authn-authz/controlling-access.md

```diff
@@ -61,9 +61,18 @@ While Kubernetes uses `usernames` for access control decisions and in request lo
 it does not have a `user` object nor does it store usernames or other information about
 users in its object store.
 
+## Prioritization and fairness
```

Comment:
Is prioritization relevant to access control?

Reply:
No.  Prioritization and fairness are both about how authorized requests are handled.